### ISSUE TITLE
Move cache to tmpfs (0.80/Home Assistant)

### ISSUE THREAD

**Issue by Ysbrand** [blakeblackshear/frigate#349]
I'm running 0.80 in HassOS installed via the addon repository.

Currently cache is living in /tmp:
`root@ccab4aaf-frigate-beta:/tmp/cache`

But /tmp is not a separate filesystem:
```
root@ccab4aaf-frigate-beta:/# df -h /tmp
Filesystem      Size  Used Avail Use% Mounted on
overlay         220G   29G  180G  14% /
```

Suggestion to mount cache as a tmpfs in /tmp. Obviously with some memory limitation.
Might be possible to do this as part of the frigate startup, in that case it could be even configured in the configuration file.

And just to be clear /tmp/cache is also NOT a separate filesystem:
```
df -h /tmp/cache
Filesystem      Size  Used Avail Use% Mounted on
overlay         220G   35G  174G  17% /
```

**Comment by tunip** [comment]
tmpfs mountpoint in 0.8 is /tmp/cache (not /tmp). You could try `df -h /tmp/cache`

```
root@ff839794c235:/opt/frigate# df -h /tmp/cache
Filesystem      Size  Used Avail Use% Mounted on
tmpfs           954M  100M  855M  11% /tmp/cache
```

**Comment by Ysbrand** [comment]
After updating to Beta 2 thinsg are now slightly different:

```
frigate.app                    INFO    : Creating tmpfs of size 256m
mount: /tmp/cache: mount point does not exist.
frigate.app                    ERROR   : Failed to create tmpfs, error code: 8192
frigate.app                    INFO    : Creating directory: /tmp/cache
```

Looking at the code, it appears to me that the order is wrong, the directory will need to be created before issuing the:
```
mount -t tmpfs -o size=256m tmpfs /tmp/cache
mount: /tmp/cache: mount point does not exist.
mkdir /tmp/cache
mount -t tmpfs -o size=256m tmpfs /tmp/cache
df -h /tmp/cache
Filesystem      Size  Used Avail Use% Mounted on
tmpfs           256M   43M  214M  17% /tmp/cache

```

**Comment by msekoranja** [comment]
You are right, see also #412

**Comment by Ysbrand** [comment]
I can confirm that this issue is fixed in 0.8.0 Beta 3:

```
frigate.app                    INFO    : Creating directory: /tmp/cache
frigate.app                    INFO    : Creating tmpfs of size 256m
```
